observability/logging/logging-6.0/log60-cluster-logging-support.adoc

```diff
@@ -0,0 +1,47 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="log60-cluster-logging-support"]
+include::_attributes/common-attributes.adoc[]
+= Support
+:context: log60-cluster-logging-support
+
+toc::[]
+
+include::snippets/logging-supported-config-snip.adoc[]
+include::snippets/logging-compatibility-snip.adoc[]
+include::snippets/log6x-loki-statement-snip.adoc[]
+
+{logging-uc} {for} is an opinionated collector and normalizer of application, infrastructure, and audit logs. It is intended to be used for forwarding logs to various supported systems.
+
+{logging-uc} is not:
+
+* A high scale log collection system
+* Security Information and Event Monitoring (SIEM) compliant
```

Comment:
Can  you add:

A "bring your own" (BYO) log collector configuration

Reply:
done